src/clojure/nrepl/misc.clj

```diff
@@ -48,3 +53,23 @@
                                                (-> session meta :id)
                                                session)}))]
     (merge basis response)))
+
+(defn requiring-resolve
```

Comment:
I think internally we have at least 5 definitions of this here and there, so you can also replace those once this PR is merged. It’d be nice if we unified them.

Reply:
Will do.